components/more_channels/index.js

```diff
@@ -15,4 +19,14 @@ function mapDispatchToProps(dispatch) {
     };
 }
 
-export default connect(null, mapDispatchToProps)(MoreChannels);
+function mapStateToProps(state) {
+    const team = getCurrentTeam(state) || {};
+
+    return {
+        channels: getOtherChannels(state) || [],
+        teamId: team.id,
+        teamName: team.name,
+    };
+}
```

Comment:
Non-blocking comment but it would be nice to throw this function above `mapDispatchToProps` for consistency